owner-onboarding-server/src/main.rs

```diff
@@ -138,10 +152,51 @@ async fn report_to_rendezvous(udt: OwnerServiceUDT) -> Result<()> {
     Ok(())
 }
 
+async fn check_registration_window(udt: &OwnerServiceUDT) -> Result<()> {
+    let now_plus_window =
+        time::OffsetDateTime::now_utc().unix_timestamp() + (udt.ov_re_registration_window as i64);
+    // these are the ovs whose registration time will end and we need to
```

Comment:
Extra Nitty: It seems like all other comments are capitalized, does it make sense to capitalize these too?

Reply:
I usually capitalise when I'm describing a function or global parameters or struct fields, but I don't / don't care when  I'm making a comment inside a function